client/network.go

```diff
@@ -45,28 +40,16 @@ func (cli *NetworkCli) CmdNetwork(chain string, args ...string) error {
 // CmdNetworkCreate handles Network Create UI
 func (cli *NetworkCli) CmdNetworkCreate(chain string, args ...string) error {
 	cmd := cli.Subcmd(chain, "create", "NETWORK-NAME", "Creates a new network with a name specified by the user", false)
-	flDriver := cmd.String([]string{"d", "-driver"}, "bridge", "Driver to manage the Network")
-	cmd.Require(flag.Min, 1)
+	flDriver := cmd.String([]string{"d", "-driver"}, "", "Driver to manage the Network")
+	cmd.Require(flag.Exact, 1)
 	err := cmd.ParseFlags(args, true)
 	if err != nil {
 		return err
 	}
-	if *flDriver == "" {
-		*flDriver = defaultDriverType
-	}
 
 	// Construct network create request body
 	ops := make(map[string]interface{})
 	nc := networkCreate{Name: cmd.Arg(0), NetworkType: *flDriver, Options: ops}
-
```

Comment:
I am confused why we can't have the cli code take care of the default network.
At the end, the requirement for this comes from the user experience of human user not to have to specify the network if he plans to only work with one network. And human user case is cli only.

I don't believe the rest api has to provide this. If you take the time to write rest client api, adding or not the network field in the POST requests makes no difference.

Same for cli scripting. If you take the time to write a cli script, what difference it makes to specify or not the network name in your script?